src/logic/Convertor.cpp

```diff
@@ -4,7 +4,7 @@
 
 #include "Convertor.h"
 
-TaskDTO Convertor::convert(std::weak_ptr<FullTask> ft){
+TaskDTO Convertor::getTaskDTO(std::weak_ptr<FullTask> ft){
```

Comment:
probably, convert name was better.
Consider renaming it to convertToDTO

Reply:
take a look at Convertor, i added one more function which convers data, so i need to distinguish these functions

Reply:
take a look at Convertor, i added one more function which convers data, so i need to distinguish these functions